thank-you.php

```diff
@@ -0,0 +1,22 @@
+<?php
+    require_once __DIR__.'/_backend/preload.php';
+    $page['title'] = 'Thank You for downloading';
+    $page['theme-color'] = '#3E4E54';   
+    
+    include $template['header'];
+    include $template['alert'];
+?>
+
+<section class="hero">
+    <div class="grid">
+    <div>
+        <img src="<?=$sitewide['root'];?>images/brand/logomark.png" alt="logomark">
```

Comment:
We tend to not use PHP shorthand. Please change this to `<?php echo $sitewide['lang-root']; ?>`